sk-usbhid.c

```diff
@@ -686,7 +686,7 @@ sk_enroll(uint32_t alg, const uint8_t *challenge, size_t challenge_len,
 		goto out;
 	}
 	*enroll_response = NULL;
-	memset(user_id, 0, sizeof(user_id));
+	arc4random_buf(user_id, sizeof(user_id));
```

Comment:
This modification was requested by @akshayku. I have pushed 1e3d6d4a to put it under `#ifdef WINDOWS`.

Reply:
This is needed so that existing credentials are not overwritten in FIDO Authenticators. I believe this should be generic to all the platforms but I am fine if we want to put it under #ifdef WINDOWS block.

Reply:
Please add a comment for future reference.

Reply:
Addressed in 78920431.